libsolutil/CommonData.cpp

```diff
@@ -38,7 +39,7 @@ static char const* lowerHexChars = "0123456789abcdef";
 
 }
 
-string dev::toHex(uint8_t _data, HexCase _case)
+string solidity::util::toHex(uint8_t _data, HexCase _case)
```

Comment:
``util`` alone should be enough, shouldn't it?

Reply:
That's why I don't like doing it like this at all. I prefer clear association of symbols to scopes. With doing `namespace X { Y(); .... }`  you know it's `X::Y();`. With the above, sure, it *should* be sufficient to just do `util::toHex()....`, as CC is first looking up `util` and then checks if it is implementing a forward declared `toHex`. It may be clear too, but it apparently isn't too obvious (to me at least). I'm not sure we want to shortcut here.

Reply:
Maybe we should put up opening instead of importing namespaces in implementations in general... generally, I also prefer that and I could imagine that the main motivation against that was that it needs multiple lines for both opening and closing which is no longer the case with C++17...